### PR TITLE
Generic SQL fetch-incidents

### PR DESCRIPTION
**Pull request by rshunim**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-921)

## Description
Add support for fetch incidents (Generic SQL integration)

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by yuvalbenshalom — CHANGES_REQUESTED** [review]
see my comments

File: Packs/GenericSQL/Integrations/GenericSQL/GenericSQL_test.py [hunk 1]
```diff
@@ -253,3 +253,134 @@ def test_mysql_integration():
      {'arg1': 'value1', 'arg2': 'value2', 'driver': 'ODBC Driver 17 for SQL Server'})])
 def test_parse_connect_parameters(connect_parameters, dialect, expected_response):
     assert Client.parse_connect_parameters(connect_parameters, dialect) == expected_response
+
+
```
[the remark this answers is not in the document]
Comment (rshunim) [reply Packs/GenericSQL/Integrations/GenericSQL/GenericSQL_test.py:257]: it's not mine

[the remark this answers is not in the document]
Comment (rshunim) [reply Packs/GenericSQL/Integrations/GenericSQL/GenericSQL_test.py:257]: In general I'll add a few unit tests for the de duplication

**Review by yuvalbenshalom — APPROVED** [review]

**Comment by ShirleyDenkberg** [comment]
@yuvalbenshalom Doc review completed.

**Pull request merged**